Lib/test/test_inspect/test_inspect.py

```diff
@@ -29,6 +28,12 @@
 import warnings
 import weakref
 
+try:
+    import _pickle
+    MISSING_C_PICKLE = False
+except ImportError:
+    MISSING_C_PICKLE = True
```

Comment:
```suggestion
try:
    import _pickle
except ImportError:
    _pickle = None
```

This pattern is used more often in tests, as shown in the line below, it removes the need for a variable and makes the codebase more consistent.

Reply:
With this, use `@unittest.skipUnless(_pickle, "requires _pickle module")` as the test decorator.

Or to pick a different pattern in use in this module, use `_pickle = import_helper.import_module('_pickle')` in the test (see some examples that use `_testcapi`).

As another option, is there a particular reason to use `_pickle`?  Can/should something from `_testcapi` be used instead?